xbmc/settings/dialogs/CMakeLists.txt

```diff
@@ -1,13 +1,9 @@
-set(SOURCES GUIDialogAudioDSPManager.cpp
```

Comment:
Removing `GUIDialogAudioDSPManager.cpp` from the build files causes the linking error.
After readding it (and commenting out some `CServiceBroker::GetADSP()`) it links again without errors.